src/OnlySpans.PolyLeads.Api.Schema/Payloads/Documents/RecognitionStatus.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace OnlySpans.PolyLeads.Api.Schema.Payloads.Documents;
+
+[GraphQLDescription(
+    """
+    Статус распознавания текста:
+    0 - неизвестен
+    1 - в очереди
+    2 - выполняется
+    3 - текст успешно распознан
+    4 - при распозновании текста произошла ошибка
+    """)]
+public enum RecognitionStatus
+{
+    Unknown,
+    Queued,
+    Processing,
```

Comment:
в enum всегда проставляй числа

```suggestion
    Unknown = 0,
    Queued = 1,
    Processing = 2,
```